RevenueCatUI/Templates/Example1Template.swift

```diff
@@ -35,21 +41,37 @@ struct Example1Template: TemplateViewType {
     var body: some View {
         switch self.data {
         case let .success(data):
-            Example1TemplateContent(data: data)
+            Example1TemplateContent(data: data, introEligibility: self.introEligibility)
+                .task(id: self.package) {
+                    if let package = self.package {
+                        self.introEligibility = await self.introEligibilityChecker.eligibility(
+                            for: package.storeProduct
+                        )
+                    }
+                }
+
         case let .failure(error):
-            DebugErrorView(error)
+            // Fix-me: consider changing this behavior once we understand
+            // how unlikely we can make this to happen thanks to server-side validations.
+            DebugErrorView(error, releaseBehavior: .emptyView)
         }
     }
 
+    private var package: Package? {
+        return (try? self.data.get())?.package
```

Comment:
I miss `Result.value`. Can't wait to be able to change all these helpers to `package`.